src/constants/index.ts

```diff
@@ -126,6 +126,15 @@ export const TOKEN_DATA = {
   [Network.Mainnet]: ETHER_TOKEN,
 };
 
+const GAS_LIMITS = {
```

Comment:
With multicall, we theoretically may provide a gas estimate that exceeds the block limit. Therefore, knowing what the block limit is enables us to ensure that from the front end, an estimate never exceeds it.